NetKAN/FusTek-SharedAssets.netkan

```diff
@@ -0,0 +1,19 @@
+{
+    "spec_version"   : "1.4",
+    "identifier"     : "FusTek-SharedAssets",
+    "$kref"          : "#/ckan/github/sumghai/FusTek_SharedAssets",
```

Comment:
Checking the repository, the netkan bot will fail to build until there's at least one ZIP in a non-prerelease release.